spec/lib/blacklight/document/cache_key_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'spec_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.